tensorrt_llm/llmapi/mpi_session.py

```diff
@@ -263,28 +264,54 @@ class RemoteMpiCommSessionClient():
     RemoteMpiCommSessionClient is a variant of MpiCommSession that is used to connect to a remote MPI pool.
     '''
 
-    def __init__(self, addr: str, hmac_key: Optional[bytes] = None):
+    def __init__(
+        self,
+        *,
+        addr: Optional[tuple[str, Optional[bytes]]] = None,
+        addr_file: Optional[str] = None,
+    ):
+        '''
+        Args:
+            addr: The address of the remote MPI pool.
+            addr_file: The file to write the address of the remote MPI pool to share with other intra-node processes.
+        '''
         # FIXME: this is a hack to avoid circular import, resolve later
         from tensorrt_llm.executor.ipc import ZeroMqQueue
-        self.addr = addr
+        self._is_shutdown = False
         print_colored_debug(
             f"RemoteMpiCommSessionClient connecting to {addr}\n", "yellow")
-        self.queue = ZeroMqQueue((addr, hmac_key),
-                                 is_server=False,
-                                 use_hmac_encryption=bool(hmac_key))
-        self._is_shutdown = False
+        self.queue = ZeroMqQueue(addr, is_server=True, use_hmac_encryption=True)
```

Comment:
Why the "is_server" is changed from false to true?
Its "client" process here right?